pkg/arvo/sys/vane/ames.hoon

```diff
@@ -2055,6 +2059,10 @@
           ?:  for
             event-core
           (try-next-sponsor sponsor.peer-state)
+        ::  if forwarding, route must not be stale
+        ::
+        ?:  &(for (lth last-contact.qos.peer-state (add now ~h1)))
```

Comment:
I think this should call `+sub`.

Reply:
ah yes, this line was designed for ships that move backward through time, like positrons